src/test/java/teammates/test/cases/logic/InstructorsLogicTest.java

```diff
@@ -728,4 +730,25 @@ private void verifySameInstructor(InstructorAttributes instructor1, InstructorAt
         assertEquals(instructor1.email, instructor2.email);
     }
 
+    private void testGetCoOwnersForCourse() {
+        ______TS("Verify co-owner status of generated co-owners list");
+        String courseId = "idOfTypicalCourse1";
+        List<InstructorAttributes> generatedCoOwners = instructorsLogic.getCoOwnersForCourse(courseId);
+        for (InstructorAttributes generatedCoOwner : generatedCoOwners) {
+            assertTrue(generatedCoOwner.hasCoownerPrivileges());
+        }
+
+        ______TS("Verify all co-owners present in generated co-owners list");
+
+        List<InstructorAttributes> instructors = instructorsLogic.getInstructorsForCourse(courseId);
```

Comment:
@leeyimin I think I found a solution. Instead of the sort, which is a little cumbersome, I decided to change the value being checked for membership from an `InstructorAttributes` object to a String of the Instructor's email ID. So now, `List1.containsAll(List2) && List2.containsAll(List1)` works as it is now a `List<String>` object. Further details in change summary comment below.